tests/sqllogictests/suites/base/05_ddl/05_0004_ddl_create_user

```diff
@@ -1,53 +1,59 @@
 statement ok
-DROP USER IF EXISTS 'test-a'@'localhost'
+DROP USER IF EXISTS 'test-a'
 
 statement ok
-DROP USER IF EXISTS 'test-b'@'localhost'
+DROP USER IF EXISTS 'test-b'
 
 statement ok
-DROP USER IF EXISTS 'test-c'@'localhost'
+DROP USER IF EXISTS 'test-c'
 
 statement ok
-DROP USER IF EXISTS 'test-d@localhost'
+DROP USER IF EXISTS 'test-d'
 
 statement ok
 DROP USER IF EXISTS 'test-e'
 
+statement error 1005
+CREATE USER 'test-a'@'127.0.0.1' IDENTIFIED BY 'password'
+
+statement error 1005
+CREATE USER 'test-a'@'%' IDENTIFIED BY 'password'
```

Comment:
I guess keeping this statement compatibility is ok? just allow '%' and returns error on other values